tools/docker-images/clp-execution-base-ubuntu-noble/build.sh

```
@@ -0,0 +1,38 @@
+if [ "$arch" = "x86_64" ]; then
+    arch_name="x86"
+elif [ "$arch" = "aarch64" ]; then
+    arch_name="arm64"
+else
+    echo "Error: Unsupported architecture - $arch"
+    exit 1
+fi
+
+build_cmd=(
+    docker build
+    --tag clp-execution-${arch_name}-ubuntu-noble:dev
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)